cmd/cluster/aws/create.go

```diff
@@ -178,7 +178,7 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 		}
 	}
 
-	tagMap, err := util.ParseAWSTags(opts.AWSPlatform.AdditionalTags)
+	tagMap, err := util.ParseTags(opts.AWSPlatform.AdditionalTags)
```

Comment:
Regarding the forbidden tag comments I would add a validation (maybe I would convert that function to a method based in the provider) to check the tags by provider.

Reply:
I think we would end up with a lot of duplicate code, but it could be worth looking into a new function to do it . Wdyt ?

Reply:
I think if we were to add any tags validation, we would add it in the backend rather than here.